src/main/java/seedu/address/model/activity/Activity.java

```diff
@@ -87,6 +109,48 @@ public Title getTitle() {
         return title;
     }
 
+    /**
+     * Gets the transfer matrix. Every (i, j) entry reflects how much i receives
+     * from j. Negative amounts means i has to give j money.
+     */
+    public ArrayList<ArrayList<Double>> getTransferMatrix() {
+        simplifyExpenses();
+        return transferMatrix;
+    }
+
+    /**
+     * Invite people to the activity.
+     * @param people The people that will be added into the activity.
+     */
+    public void invite(Person ... people) {
```

Comment:
Is this invite method being used somewhere? Also, is there a reason on why this endpoint isn't updating debtMatrix/transferMatrix?

Reply:
it does, it uses the invite below this one